# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/moz.build
# Commit: de6a26d3e3df
# Full Hash: de6a26d3e3dfd5d09121f11ae2b09903c780179b
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-17 17:00:49
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 4: Implement the third-party cookie blocking excpetion in CookieService. r=cookie-reviewers,baku,valentin
#   
#   The patch integrates the third-party cookie blocking excpetion in
#   CookieService.
#   
# ==============================================================================

diff -r b2934cae3c23 -r de6a26d3e3df netwerk/cookie/moz.build
--- a/netwerk/cookie/moz.build	Tue Dec 17 08:57:30 2024 +0000
+++ b/netwerk/cookie/moz.build	Tue Dec 17 08:57:31 2024 +0000
@@ -34,6 +34,7 @@
     "CookieServiceChild.h",
     "CookieServiceParent.h",
     "CookieStorage.h",
+    "ThirdPartyCookieBlockingExceptions.h",
 ]
 UNIFIED_SOURCES += [
     "Cookie.cpp",
@@ -48,6 +49,7 @@
     "CookieServiceChild.cpp",
     "CookieServiceParent.cpp",
     "CookieStorage.cpp",
+    "ThirdPartyCookieBlockingExceptions.cpp",
 ]
 XPCSHELL_TESTS_MANIFESTS += [
     "test/unit/xpcshell.toml",